src/memex/presenters.py

```diff
@@ -106,7 +128,6 @@ def asdict(self):
             'tags': self.tags,
             'tags_raw': self.tags,
             'group': self.annotation.groupid,
-            'permissions': self.permissions,
```

Comment:
Can we add a FIXME next to the relevant item in the mapping so we remember to remove it?